db/migrations/0003_alter_player_race_alter_skill_race.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2024-11-11 15:51
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_alter_player_created_at_alter_player_guild'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race'),
+        ),
+        migrations.AlterField(
+            model_name='skill',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race'),
```

Comment:
Consider adding a `related_name` for the `race` ForeignKey in the `Skill` model if reverse querying is needed. The previous `related_name` was 'skills', which was appropriate.